references/classification/train.py

```diff
@@ -229,12 +229,16 @@ def main(args):
 
     criterion = nn.CrossEntropyLoss(label_smoothing=args.label_smoothing)
 
-    if args.norm_weight_decay is None:
-        parameters = model.parameters()
-    else:
-        param_groups = torchvision.ops._utils.split_normalization_params(model)
-        wd_groups = [args.norm_weight_decay, args.weight_decay]
-        parameters = [{"params": p, "weight_decay": w} for p, w in zip(param_groups, wd_groups) if p]
+    custom_keys_weight_decay = None
+    if hasattr(model, "no_weight_decay_keys"):
+        custom_keys_weight_decay = {k: 0.0 for k in model.no_weight_decay_keys()}
+    parameters = torchvision.ops._utils.set_weight_decay(
+        model,
+        args.weight_decay,
+        norm_weight_decay=args.norm_weight_decay,
+        bias_weight_decay=args.bias_weight_decay,
```

Comment:
Bias was removed if I understand correctly, so this won't work.

Reply:
yes, bias can be handled in the  `custom_keys`, eg: 
`
custom_keys_weight_decay = [("bias", 0.0)]
`